### PR TITLE
Removing converters - Item,ItemDetails,Image

### PR DESCRIPTION
**Pull request by pawelKapl**

### CODE REVIEW

File: database/apid/src/main/java/com/example/javaopencommerce/item/ItemEntity.java [hunk 1]
```diff
@@ -20,6 +24,21 @@
 @EqualsAndHashCode
 public class ItemEntity {
 
+    static ItemEntity fromSnapshot(ItemSnapshot itemSnapshot) {
+        ImageEntity image = ImageEntity.builder()
+            .id(itemSnapshot.getImage().getId())
```
Comment (Augustyn) [inline database/apid/src/main/java/com/example/javaopencommerce/item/ItemEntity.java:29]: it's not null safe operation. Consider extracting image to variable:
```suggestion
ImageSnapshot img = Optional.ofNullable(itemSnapshot).map(ItemSnapshot::getImage).orElse(ImageSnapshot.builder.build();
ImageEntity image = ImageEntity.builder()
.id(img.getId())
```

File: database/apid/src/main/java/com/example/javaopencommerce/item/ItemEntity.java [hunk 2]
```diff
@@ -29,4 +48,18 @@
     @Builder.Default
     private List<ItemDetailsEntity> details = new ArrayList<>();
 
+    Item toItemModel() {
+        List<ItemDetails> detailsModels = details.stream()
```
Comment (Augustyn) [inline database/apid/src/main/java/com/example/javaopencommerce/item/ItemEntity.java:52]: It's not null-safe as well.
```suggestion
        List<ItemDetails> detailsModels = Optional.ofNullable(details).orElse(Collections.emptyList()).stream()
```

**Review by Augustyn — APPROVED** [review]

File: database/apid/src/main/java/com/example/javaopencommerce/item/ItemEntity.java (hunk 1 above)
Comment (pawelKapl) [reply database/apid/src/main/java/com/example/javaopencommerce/item/ItemEntity.java]: Fixed!

File: database/apid/src/main/java/com/example/javaopencommerce/item/ItemEntity.java (hunk 2 above)
Comment (pawelKapl) [reply database/apid/src/main/java/com/example/javaopencommerce/item/ItemEntity.java]: Fixed!

**Pull request merged**